config.sample.yaml

```diff
@@ -50,6 +50,9 @@ tre:
   firewall_sku: Standard
   app_gateway_sku: Standard_v2
 
+  # Set to true if TreAdmins should be able to assign and de-assign users to workspaces via the UI
+  user_management_enabled: true
```

Comment:
What will happen if this is true and `auto_workspace_group_creation` is false (which is the default)?

Reply:
Will default to false for now